tests/pipelines/fgcmMakeLutHsc.yaml

```diff
@@ -0,0 +1,5 @@
+description: Make fgcmcal look-up table
+instrument: lsst.obs.subaru.HyperSuprimeCam
+tasks:
+  fgcmMakeLut:
+    class: lsst.fgcmcal.fgcmMakeLut.FgcmMakeLutTask
```

Comment:
Also I should make note that if you dont have any other fields for the class you can just do:
```
fgcmMakeLut: lsst.fgcmcal.fgcmMakeLust.FgcmMakeLutTask
```